jest-config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  testEnvironment: "jsdom",
+  moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
+  testMatch: ["**/jest/**/?(*.)+(spec|test).[jt]s?(x)"],
+  testPathIgnorePatterns: ["victory-native"],
+  transform: {
+    "^.+\\.(js|jsx)$": [
+      "babel-jest",
+      { configFile: "./babel-native-config.js" }
```

Comment:
@ryan-roemer I split this out into separate configs again so we can run both `nps jest` and `nps jest.native` to run either the `victory-native` tests or the rest of the victory tests.

However, I'm running into some issues with the babel config. It works with the babel native config, but if I change this line to `babel-config.js` I get some cryptic errors. This file just has the same `@babel/react` and `@babel/env` presets that the `.babelrc` files have, but there's some piece of config in the metro config that must be required for `babel-jest` and I haven't figured out yet what it is. Any ideas about what could be wrong here?

Reply:
@becca-bailey i don't have full context here so this may not be the right answer at all; but it sounds like we may want the RN Babel preset here?

https://github.com/facebook/react-native/blob/main/template/babel.config.js
https://github.com/facebook/metro/tree/main/packages/metro-react-native-babel-preset